docs/docs-components/data/components-new.js

```diff
@@ -0,0 +1,2606 @@
+// @flow strict
```

Comment:
Note that this is a different file from `docs/docs-components/data/components.js`. That file remains untouched, so no changes here will be reflected in the docs yet.